Editor/Module Management/ModuleList.cs

```diff
@@ -46,14 +56,18 @@ public static class ModuleList
             version = "4.0.2"
         };
 
+        /// <summary>
+        ///     Get installed modules from Modules list.
+        /// </summary>
+        /// <returns>A <c>Dictionary&lt;string, string&gt;</c> in the format of <c>Dictionary&lt;name, version&gt;</c>. </returns>
```

Comment:
Is this line correct? Should we use such symbols in the code?

Reply:
It's correct, perhaps unnecessary but it allows you to show the type of dictionary (including the < and > symbols in the code docs 
![image](https://user-images.githubusercontent.com/7085672/211274112-4d8ca387-6889-49a0-a4e7-e3c5c1d9dc70.png)

Reply:
I've updated to this now actually 
![image](https://user-images.githubusercontent.com/7085672/211275457-6d95cd55-cfa5-409f-8453-dffd8cc88c3d.png)